Cabal/Distribution/Types/Library.hs

```diff
@@ -9,22 +9,24 @@ module Distribution.Types.Library (
     libModules,
 ) where
 
-import Prelude ()
 import Distribution.Compat.Prelude
+import Prelude ()
 
+import Distribution.ModuleName
 import Distribution.Types.BuildInfo
+import Distribution.Types.LibraryVisibility
 import Distribution.Types.ModuleReexport
 import Distribution.Types.UnqualComponentName
-import Distribution.ModuleName
 
 import qualified Distribution.Types.BuildInfo.Lens as L
 
 data Library = Library
     { libName           :: Maybe UnqualComponentName
     , exposedModules    :: [ModuleName]
     , reexportedModules :: [ModuleReexport]
-    , signatures        :: [ModuleName]   -- ^ What sigs need implementations?
-    , libExposed        :: Bool           -- ^ Is the lib to be exposed by default?
+    , signatures        :: [ModuleName]       -- ^ What sigs need implementations?
+    , libExposed        :: Bool               -- ^ Is the lib to be exposed by default? (i.e. whether its modules available in GHCi for example)
+    , libVisibility     :: LibraryVisibility  -- ^ Whether this multilib can be dependent from outside.
```

Comment:
I think the comment should be "[...]can be depended on from outside"

Reply:
We could make it even clearer what is meant by "outside" by saying something like "... can be depended on from other packages"